test/algorithms/set_operations/difference/difference.cpp

```diff
@@ -624,7 +624,7 @@ int test_main(int, char* [])
 #if defined(BOOST_GEOMETRY_TEST_FAILURES)
     // Not yet fully tested for float and long double.
     // The difference algorithm can generate (additional) slivers
-    BoostGeometryWriteExpectedFailures(10, 11, 24, 14);
+    BoostGeometryWriteExpectedFailures(10, 11, 24, 15);
```

Comment:
_(this was already)_

Reply:
This means that there is no extra error here?

Reply:
Nope, these statements are mainly for myself, during transition of removing rescaling. This is one for `long double`. I probably didn't update it correctly last time. It's not updated automatically.
For this PR, I commented the code (updating the fraction) and it was still there - so it's not this PR for sure.

Main purpose is to see if we get there. For set operations, the version without rescaling is on par, or better. For buffer it's still not good enough so I'm concentrating on those cases.